### PR TITLE
Loadable solutions

### PR DESCRIPTION
**Pull request by VK432**

### CODE REVIEW

**Comment by CelineBoudier** [comment]
looks like codeclimate is complaining!

**Comment by VK432** [comment]
All codeclimate issues, are from lines being too long, in my dictionary of blockly solutions, I think these can be ignored. I think making them multiple lines would make the file less clear!

**Comment by PhydeauxLeChien** [comment]
you can stop codeclimate complaining about line length: see ocadotechnology/codeforlife-portal#190

File: game/views/level.py [hunk 1]
```diff
@@ -325,6 +327,31 @@ def save_workspace(request, workspaceID=None):
...
+        else:
+            workspace.python_contents = solutions[levelName]
+            workspace.blockly_enabled = False
+            workspace.python_enabled = True
+
+        return HttpResponse(json.dumps({'contents': workspace.contents,
+                                        'python_contents': workspace.python_contents}),
+                            content_type='application/json')
+
+    return HttpResponse(json.dumps(''), content_type='application/json')
```
Comment (joshuablake) [inline game/views/level.py]: Probably wants to be a 404

**Comment by CelineBoudier** [comment]
Can you squash some of the duplicate commits when the tests have successfully passed? :) Thanks!

**Comment by VK432** [comment]
Squashed the commits, now passing the tests also!

**Pull request merged**